h2o-r/h2o-package/R/frame.R

```diff
@@ -3672,14 +3672,16 @@ h2o.merge <- function(x, y, by=intersect(names(x), names(y)), by.x=by, by.y=by,
 #'
 #' @param x The H2OFrame input to be sorted.
 #' @param \dots The column names to sort by.
+#' @param sortDir The sorting direction with ASC (ascending and default) or DESC (descending).
 #'
 #' @export
-h2o.arrange <- function(x, ...) {
+h2o.arrange <- function(x, ..., sortDir="ASC") {
```

Comment:
A few issues with the R interface here:

- We should not use camel case in R (or Python) argument names.  
- There are already conventions in H2O and in R for "ascending" so "ASC" is not the best value to use here.

In R, ideally, this would work like `dplyr::arrange()`, where it will choose ascending by default and if you wrap a variable name in `desc()`, it will change the order.  Can you do that?  Example:
```
arrange(mtcars, cyl, disp)
arrange(mtcars, desc(disp))
```